hub/core/chunk_engine/chunker.py

```diff
@@ -86,7 +86,10 @@ def join_chunks(chunks: List[bytes], start_byte: int, end_byte: int) -> bytes:
     Returns:
         bytes: The chunks joined as one bytes object.
     """
+    if len(chunks) == 1:
+        return memoryview(chunks[0])[start_byte:end_byte]
 
+    # TODO make below case efficient using memoryview
```

Comment:
@McCrearyD was this already fixed in one of your PRs?

Reply:
yeah but it's not merged with release 2.0 yet (it's currently in the append PR)